ci/jenkins/test.sh

```diff
@@ -336,7 +336,7 @@ function run_e2e {
 
     set +e
     mkdir -p `pwd`/antrea-test-logs
-    go test -v github.com/vmware-tanzu/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs -timeout=50m --prometheus
+    go test -v github.com/vmware-tanzu/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs -timeout=100m --prometheus
```

Comment:
Is the timeout increase necessary? If yes, can you provide more accurate values?

same below

Reply:
Yes, it's necessary. For dual-stack, it takes 1h37min. For ipv6-only it takes 1h9min.

Reply:
ok, but the Kind ones?

Reply:
Kind doesn't need that long. But 55min sometime will cause timeout, I put 100m there to unify it with the Jenkins one. And I thought use a longer timeout to make sure it won't timeout won't have a negative effect？
But yeah, if it's not good to use a too-long timeout, I can change it to a more accurate value after the current test complete and show the time it took.

Reply:
Seems Kind tests take up to 57m. Do you want me to change it to 60m or something?

Reply:
70m would be fine to have some buffer

Reply:
Done.